packages/icon-button/icon-button.scss

```diff
@@ -0,0 +1,12 @@
+@import "~@fremtind/jkl-core/screen-reader.scss";
+
+.jkl-icon-button {
+    border: none;
```

Comment:
trenger full reset for å bli kvitt outlines osv i feks firefox

Reply:
Jeg har brukt reset-outline. Er det ok?
3c5fbdd4eab20a8410c47b09e3eccd9b7fc73318

Reply:
Det fungerer 😊 Jeg holder på med en reset som er litt mer spesifikk for knapper, men `reset-outline` er topp så lenge!